pkg/rpm/augment_rpm_files_install.py

```diff
@@ -22,10 +22,14 @@
 import sys
 import json
 
-# NOTE: Keep this in sync with the same variable in rpm.bzl
+# NOTE: Keep those two in sync with the same variables in rpm_pfg.bzl
 _INSTALL_FILE_STANZA_FMT = """
-install -d %{{buildroot}}/$(dirname {1})
-cp {0} %{{buildroot}}/{1}
+install -d "%{{buildroot}}/$(dirname '{1}')"
+cp '{0}' '%{{buildroot}}/{1}'
+""".strip()
+
+_FILE_MODE_STANZA_FMT = """
+{0} "{1}"
```

Comment:
Should {0} be quoted?

Reply:
I don't think so. The `{0}` argument is replaced with `%attr()` parameter and I don't see anything about quoting it ([source](http://ftp.rpm.org/max-rpm/s1-rpm-anywhere-specifying-file-attributes.html)):

> The %attr macro has the following format:
>
> `%attr(<mode>, <user>, <group>) <file>`
>

Some valid examples are:
> ```
> %files 
> %attr(-, root, root) %doc README
> %attr(4755, root, root) /usr/local/bin/cdp
> ```

The only scenario I can think of is quotting user or group, assuming it contains whitespaces.